lib/_http_client.js

```diff
@@ -66,9 +66,10 @@ function isInvalidPath(s) {
 }
 
 function validateHost(host) {
-  if (host !== undefined && host !== null && typeof host !== 'string') {
+  // eslint-disable-next-line eqeqeq
+  if (host != undefined && typeof host !== 'string') {
     throw new TypeError(
-      'hostname must either be a string, undefined or null');
+      '"hostname" must either be a string, undefined or null');
```

Comment:
I was stewing over that. Not sure. I'm fine with doing so just not sure if we should